returnDetailAddendumD_test.go

```diff
@@ -73,6 +88,41 @@ func TestMockReturnDetailAddendumD(t *testing.T) {
 	}
 }
 
+// TestMockReturnDetailAddendumD creates a ReturnDetailAddendumD
+func TestMockReturnDetailAddendumDWithoutEndorsingBankItemSequenceNumber(t *testing.T) {
+	rdAddendumD := mockReturnDetailAddendumD()
```

Comment:
Should we be using the mock function here? 

```suggestion
	rdAddendumD := mockReturnDetailAddendumDWithoutEndorsingBankItemSequenceNumber()
```